doc/audit-logging-guide.md

```diff
@@ -0,0 +1,165 @@
+# Configuring JSON Log Enricher for Audit Logging on Kubernetes Nodes
+
+## Introduction
+
+This is a user guide to configure audit logging in a single node local Kubernetes cluster using the JSON log enricher feature. The same steps can be used for multi-node clusters and managed clusters as well. This guide provides step-by-step instructions for configuring audit logging and viewing the generated logs.
+
+The use case involves two personas:
+- **Auditor**: Configures the audit logging system and views the generated audit logs
+- **Cluster Administrator/Support Person**: Performs administrative tasks such as exec into pods or nodes, whose activities are being audited
+
+## Prerequisites
+
+To follow this guide, you'll need a Kubernetes cluster and a few command-line tools. We're using a single-node cluster (hack/local-up-cluster.sh) for demonstration, but the process works on any Kubernetes cluster.
+
+- **Kubernetes Cluster**: The examples use a version around v1.34.0.
+- **kubectl**: The command-line tool for interacting with your cluster. We use the feature [kubectl debug: add label for debugger pod](https://github.com/kubernetes/kubernetes/pull/131791) for easy cleanup of debug pods, so a recent version of the client is recommended.
+
+## Step 1: Install Cert-Manager and the Security-Profiles-Operator
+
+First, install cert-manager as a prerequisite for SPO's TLS certificates.
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.17.2/cert-manager.yaml
+kubectl --namespace cert-manager wait --for condition=ready pod -l app.kubernetes.io/instance=cert-manager
+```
+
+Next, install the SPO:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/security-profiles-operator/v0.10.0/deploy/operator.yaml
+```
```

Comment:
This could be linked from https://github.com/kubernetes-sigs/security-profiles-operator/blob/main/installation-usage.md#install-operator since vert manager is not required in every env.